x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -340,9 +362,11 @@ export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteIniti
   );
 
   /**
+   * @apiGroup DataFrameAnalytics
+   *
    * @api {get} /api/ml/data_frame/analytics/:analyticsId/messages Gets analytics job messages
    * @apiName GetDataFrameAnalyticsMessages
-   * @apiGroup DataFrameAnalytics
+   * @apiDescription Returns the list of messages of data frame analytics job.
```

Comment:
Nit, would change this to `Returns the list of audit messages for data frame analytics jobs.`

Reply:
https://github.com/elastic/kibana/pull/56006/commits/c29e181b546a9fd6aa8a9ff47d134ea283ebf3b2